app/helpers/Constants.js

```diff
@@ -0,0 +1,3 @@
+import { Dimensions } from 'react-native';
```

Comment:
helpers suggests "test helpers", prefer `constants`

we already have a `constants/`folder, suggest `constants/layout.js`

Reply:
Oops, obsolete comment.